packages/scrollable-video/src/components/video.js

```diff
@@ -214,6 +214,7 @@ const ForwardRefVideo = React.forwardRef(
     }, [pickedSourceSrc, setVideoError, setVideoLoading, preloadCacheType])
     return (
       <Video
+        className="scrollableVideo"
```

Comment:
add namespace in the css class, such as  `twreporter-scrollable-video`, to avoid name collision?

Reply:
Updated in https://github.com/twreporter/orangutan-monorepo/pull/110/commits/1ce4917df6b3e5763ac193de2834396dbec9aa29